www/artists/jcowin/js/slideshow.js

```diff
@@ -0,0 +1,28 @@
+var slideIndex = 1;
```

Comment:
Just to be clear: why do you use a 1-index for the 0-indexed slides list? Wouldn't it be easier to have slideIndex start at 0, and not subtract 1 from it each time you access stuff?